components/rms/slurm/SPECS/slurm.spec

```diff
@@ -94,6 +94,7 @@ Patch0: slurm.conf.example.patch
 %bcond_with lua
 %bcond_with numa
 %bcond_with pmix
+%bcond_with rsmi
```

Comment:
The repository you mentioned seems to be for RHEL. So this only needs to be enabled on RHEL builds for now.

Reply:
Yes, this is true.